packages/twenty-front/src/modules/error-handler/components/PromiseRejectionEffect.tsx

```diff
@@ -20,7 +20,7 @@ export const PromiseRejectionEffect = () => {
           },
         );
       } else {
-        enqueueSnackBar(`Error: ${event.reason}`, {
+        enqueueSnackBar(`${error.message}`, {
```

Comment:
red color and message usually already convey the fact that it's an error, and there's limited space